bundles/org.openhab.binding.yioremote/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="yioremote"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<!-- Sample Thing Type -->
+	<thing-type id="yioremotedock">
+		<label>YIOremote Dock Binding Thing</label>
+		<description>YIOremote Dock Binding Thing</description>
+		<channel-groups>
+			<channel-group id="input" typeId="input"/>
+			<channel-group id="output" typeId="output"/>
+		</channel-groups>
+		<config-description>
+			<parameter name="yiodockhost" type="text" required="true">
+				<label>Host/IP Address of the YIO Dock</label>
+				<description>Host/IP Address of the YIO Dock</description>
+			</parameter>
+			<parameter name="yiodockaccesstoken" type="text" required="true">
+				<label>Access token for the YIO Dock</label>
+				<description>Access token for the YIO Dock</description>
+				<default>0</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<channel-group-type id="input">
+		<label>Inputs</label>
+		<description>The channels used for Input</description>
+		<channels>
+			<channel id="yiodockreceiverswitch" typeId="yiodockreceiverswitch"/>
+			<channel id="yiodocksendircode" typeId="yiodocksendircode"/>
+		</channels>
+	</channel-group-type>
+	<channel-group-type id="output">
+		<label>Results</label>
+		<description>The channels used for Input</description>
+		<channels>
+			<channel id="yiodockstatus" typeId="yiodockstatus"/>
+		</channels>
+	</channel-group-type>
+	<channel-type id="yiodockreceiverswitch">
+		<item-type>Switch</item-type>
+		<label>yiodockreceiverswitch</label>
```

Comment:
Can you make this speaking, as this is displayed in the UI. Same for the description

Reply:
Done